docs/catena portal guide/CatenaPortalGuide.md

```diff
@@ -0,0 +1,95 @@
+
+
+Table of Contents
+  - [Table of Contents](#table-of-contents)
+  - [Starting Point](#starting-point)
+  - [Find Your Company](#find-your-company)
+  - [Create Technical User](#create-technical-user)
+  - [Register Connector](#register-connector)
+    
+  - [NOTICE](#notice)
+
+## Starting Point
+
+Before anything else, your company needs to registered on the Catena Portal, for that you can create an issues here: https://github.com/eclipse-tractusx/portal-frontend-registration
+
+## Find Your Company
+
+When you go to the Portal website, (https://portal.int.demo.catena-x.net/) you will find this screen:
```

Comment:
Please don't refer to the consortia specific environment, this is not sustainable

Reply:
Fixed via commit:
https://github.com/catenax-ng/tx-demand-capacity-mgmt/commit/ac8a3231eeb2ff6ad0f8f699b861bc547cf6e921

Reply:
Referencing a specific operation company is not any better. Documentation has to work with what is controlled by Eclipse Tractus-X. Tractus-X is also not forwarding anyone to a operations company.
If you are mentioning the association, please use "Catena-X" instead of just "Catena"

Reply:
Remove the link to a specific OpCo